### DISCUSSION TITLE
VDL Documentation

### DISCUSSION THREAD

**Discussion by hink084** [primefaces/community#2086] [PrimeFaces] [answered]
Does PrimeFaces still host VDL documentation?

These links are outdated at the bottom of the documentation page:
https://www.primefaces.org/documentation/

![VDL](https://github.com/primefaces/community/assets/5665720/44a109d5-d977-4463-86d8-16bd3351d91a)

The latest VDL documentation I could find is from v8.
https://www.primefaces.org/docs/vdl/8.0/

I know the API's are located on javdoc.io:
https://javadoc.io/doc/org.primefaces/primefaces/latest/index.html

My assumption is no and to just reference the User Guides, is that correct?

**Comment by melloware** [comment, marked as answer]
@hink084 good point. We should be generating it and adding it to our docs site in my opinion. Can you create an issue please?

**Reply by hink084** [reply]
https://github.com/primefaces/primefaces/issues/12079

**Reply by melloware** [reply]
Permanent new home for VDL: https://primefaces.github.io/primefaces/vdldoc/
